LayoutTests/imported/w3c/web-platform-tests/html-aam/roles-contextual.html

```diff
@@ -23,8 +23,9 @@
 
 <!-- el-aside -->
 <nav>
-  <aside data-testname="el-aside-in-section-with-name" data-expectedrole="complementary" aria-label="x" class="ex">x</aside>
+  <aside data-testname="el-aside-in-section-with-name" data-expectedrole="generic" aria-label="x" class="ex">x</aside>
   <aside data-testname="el-aside-in-section-without-name" class="ex-generic">x</aside>
+  <aside data-testname="el-aside-in-section-with-role" data-expectedrole="complementary" class="ex" role="complementary">x</aside>
```

Comment:
cc @cookiecrook 

Does this new testcase seem right based on your knowledge of the spec?

Reply:
There was a discussion about this: https://bugs.chromium.org/p/chromium/issues/detail?id=1459657#c12

As I undertand, when the aside element explicitly has the role="complementary", it should retain this role regardless of whether it's nested within other elements.

Related spec: https://www.w3.org/TR/core-aam-1.2/#mapping_conflicts

Reply:
OK, makes sense. Because adds a testcase to a WPT, we'll need to either:

1. Use `Tools/Scripts/export-w3c-test-changes` to export the change to https://github.com/web-platform-tests/wpt (never used this script before unfortunately, so don't have a ton more guidance)
2. Make a [PR to the WPT](https://github.com/web-platform-tests/wpt) outside of this WebKit PR, then re-import the tests via `Tools/Scripts/import-w3c-tests -t web-platform-tests/html-aam`

Reply:
I've created a PR to the WPT by running ```Tools/Scripts/export-w3c-test-changes -c -g HEAD -b 235065```
https://github.com/web-platform-tests/wpt/pull/43028